plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/holographicdisplays/FunnyHologram.java

```diff
@@ -0,0 +1,22 @@
+package net.dzikoysk.funnyguilds.feature.hooks.holographicdisplays;
+
+import org.bukkit.Location;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.List;
+
+public interface FunnyHologram {
```

Comment:
Jest sens robić to jako interfejs z implementacją? Chyba, że chcemy wspierać jeszcze inne pluginy na hologramy.